src/sagas/repository.js

```diff
@@ -100,6 +100,38 @@ function* pollJobWorker(jobId, jobTypeSuccess, jobTypeFailure) {
   }
 }
 
+export function* exportSnapshot({ payload }) {
+  try {
+    const { snapshotId, snapshotName, terraUrl } = payload;
+    const response = yield call(
+      authGet,
+      `/api/repository/v1/snapshots/${payload.snapshotId}/export`,
+    );
+    const jobId = response.data.id;
+    yield put({
+      type: ActionTypes.EXPORT_SNAPSHOT_JOB,
+      payload: { data: response, jobId },
+    });
+    yield call(
+      pollJobWorker,
+      jobId,
+      ActionTypes.EXPORT_SNAPSHOT_SUCCESS,
+      ActionTypes.EXPORT_SNAPSHOT_FAILURE,
+    );
+    const snapshots = yield select(getSnapshotState);
+    const manifestJsonPath = snapshots.exportResponse.format.parquet.manifest;
+    window.open(
+      `${terraUrl}#import-data?url=${window.location.origin}&snapshotId=${snapshotId}&format=tdrexport&snapshotName=${snapshotName}&tdrmanifest=${manifestJsonPath}`,
```

Comment:
OIC...I think this is a little weird from a UX standpoint.  I think we want the user to do a second click, no?  Perhaps as a link.  My popup blocker blocked this

Reply:
window.open in general is kind of a bad idea for that reason

Reply:
Is there a way to do this in straight javascript that would allow it open automatically when the export finishes? Or do we have to do it as a second link, with an anchor component?

Reply:
The later (which what I thought we wanted)